src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,72 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+
+@Embeddable
+public class Answers {
+
+	@OneToMany(mappedBy = "question")
+	private final List<Answer> answerList;
+
+	protected Answers() {
+		this.answerList = new ArrayList<>();
+	}
+
+	private Answers(Collection<Answer> answers) {
+		this.answerList = new ArrayList<>(answers);
+	}
+
+	public static Answers of(Collection<Answer> answers) {
+		return new Answers(answers);
+	}
+
+	public static Answers of() {
+		return new Answers();
+	}
+
+	public boolean isAllSameWriter(User user) {
+		return answerList.stream()
+			.allMatch(answer -> answer.isOwner(user));
+	}
+
+	public DeleteHistories deleteAll() {
+		List<DeleteHistory> list = answerList.stream()
+			.map(Answer::delete).collect(Collectors.toList());
+		return DeleteHistories.of(list);
```

Comment:
```suggestion
                return answerList.stream()
                                .collect(collectingAndThen(toList(), DeleteHistories::of));
```

와 같은 마무리 연산자도 제공하니 활용해볼 수 있을 것 같아요!

Reply:
마무리 연산자도 제공하는군요! 감사합니다!